daemon/info.go

```diff
@@ -142,9 +142,23 @@ func (daemon *Daemon) showPluginsInfo() types.PluginsInfo {
 		pluginsInfo.Network = append(pluginsInfo.Network, nd)
 	}
 
+	pluginsInfo.Authorization = daemon.GetAuthorizationDriverList()
+
 	return pluginsInfo
 }
 
+// GetAuthorizationDriverList returns the list of plugins drivers
+// registered for authorization.
+func (daemon *Daemon) GetAuthorizationDriverList() []string {
+	pluginList := []string{}
+
+	for _, authz := range daemon.configStore.AuthZPlugins {
```

Comment:
you can directly return `daemon.configStore.AuthZPlugins` since it's already a `[]string`